ansible/roles/atmo-vnc/files/Xresources

```diff
@@ -0,0 +1,3 @@
+xterm*faceName: Ubuntu Mono
+xterm*faceSize: 12
+xterm*renderFont: true
```

Comment:
Do we have a fallback fontface for CentOS/Red Hat?

Reply:
I realize that there was testing to see that this does *not* impact CentOS / Ubuntu 14:

> CentOS and Ubuntu 14 instances are unaffected by the changes because the default terminal emulator is not XTerm. This is fine because the default terminal is good looking and has intuitive menus for customization if needed.

But I would be curious if there is a way to use _facts_ to select a fontface more universally (perhaps).